packages/runtime/container-runtime/src/summary/summarizer.ts

```diff
@@ -124,12 +125,20 @@ export class Summarizer extends TypedEventEmitter<ISummarizerEvents> implements
 		};
 
 		const resolvedContainer = await loader.resolve(request);
-		const fluidObject: FluidObject<ISummarizer> | undefined = resolvedContainer.getEntryPoint
-			? await resolvedContainer.getEntryPoint?.()
-			: // eslint-disable-next-line import/no-deprecated
-			  await requestFluidObject<FluidObject<ISummarizer>>(resolvedContainer, {
-					url: "_summarizer",
-			  });
+		let fluidObject: FluidObject<ISummarizer> | undefined;
+
+		// Older containers may not have the "getEntryPoint" API
+		// ! This check will need to stay until LTS of loader moves past 2.0.0-internal.7.0.0
+		if (resolvedContainer.getEntryPoint !== undefined) {
+			fluidObject = await resolvedContainer.getEntryPoint();
+		} else {
+			const response = await resolvedContainer.request({ url: summarizerRequestUrl });
+			if (response.status !== 200 || response.mimeType !== "fluid/object") {
```

Comment:
What do you think about leaving `requestFluidObject` usage here, and instead of removing it marking it as `@internal` or renaming it to `legacyRequestFluidObject` or something?  Not sure how many places you'll end up copying this code to cover these back-compat cases.